tests/unit/requirements.txt

```
@@ -14,6 +14,7 @@ linode_api4 ; python_version > '2.6'  # APIv4
 # requirement for the gitlab and github modules
 python-gitlab
 PyGithub
+github3.py
```

According to the package docs, `Note: This library currently works with Python 3.6+ or pypy3. For older versions, please use version 1.3.0.`. So you need something like
```suggestion
github3.py <= 1.3.0 ; python_version < '3.6'
github3.py ; python_version >= '3.6'
```